src/prog.js

```diff
@@ -37,6 +37,11 @@ export default handler => {
 		.option('--name', 'Specify name exposed in UMD builds')
 		.option('--cwd', 'Use an alternative working directory', '.')
 		.option('--sourcemap', 'Generate source map', true)
+		.option(
+			'--closure',
+			'Use the closure-compiler for minifying instead of terser',
+			true,
```

Comment:
should this be the default?

Reply:
It does seem equally as fast as terser from what I'm seeing

Reply:
Since this is a tool people universally expect to pass options to (due to no config), I think we should start out by merging this with the default still as Terser. We can quickly cut over to Closure as a new default once it's sat on master for a version cycle. I just don't want landing this PR and getting closure support published to necessitate a major version change all at once.

Reply:
That’s a fair compromise. Let’s make this an option (not default) to land, and iterate on output.

To become the default I think we need to support the custom mangle format microbundle uses (uglify’s native format). 

Also, the newlines should be addressed for consistency. (This work should likely be part of the rollup closure plugin).